homeassistant/components/darksky/sensor.py

```diff
@@ -371,7 +371,7 @@
 
 CONDITION_PICTURES = {
     "clear-day": ["/static/images/darksky/weather-sunny.svg", "mdi:weather-sunny"],
-    "clear-night": ["/static/images/darksky/weather-night.svg", "mdi:weather-sunny"],
+    "clear-night": ["/static/images/darksky/weather-night.svg", "mdi:night"],
```

Comment:
Is there a `night` icon?  Should it be `weather-night`?

Reply:
Yes. Sorry for that.

mdi:weather-night